config/auth.js

```
@@ -0,0 +1,57 @@
+    expiresIn: jwtConfig.expiresIn,
+    algorithm: jwtConfig.algorithm
+  })
+}
+
+const verifyToken = (token) => {
+  try {
+    return jwt.verify(token, jwtConfig.secret)
+  } catch (error) {
+    // Preserve the original JWT error details for better debugging
+    // The error will be handled by the error middleware which can distinguish between different JWT error types
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)